docs/install_guides/installing-openvino-yocto.md

```diff
@@ -1,40 +1,52 @@
 # Create a Yocto Image with Intel® Distribution of OpenVINO™ toolkit {#openvino_docs_install_guides_installing_openvino_yocto}
+
 This document provides instructions for creating a Yocto image with Intel® Distribution of OpenVINO™ toolkit.
 
 ## System Requirements
-Use the [Yocto Project official documentation](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#compatible-linux-distribution) to set up and configure your host machine to be compatible with BitBake.
+
+Follow the [Yocto Project official documentation](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#compatible-linux-distribution) to set up and configure your host machine to be compatible with BitBake.
 
 ## Step 1: Set Up Environment
 
-### Set Up Git Repositories
-The following Git repositories are required to build a Yocto image:
+### 1. Clone the repository.
 
-- [Poky](https://git.yoctoproject.org/poky)
-- [Meta-intel](https://git.yoctoproject.org/meta-intel/tree/README)
-- [Meta-openembedded](http://cgit.openembedded.org/meta-openembedded/tree/README)
-- <a href="https://github.com/kraj/meta-clang/blob/master/README.md">Meta-clang</a>
+```sh
+git clone https://git.yoctoproject.org/git/poky --branch langdale
+```
+
+### 2. Navigate to the "poky" folder and clone the following repositories.
+
+```sh
+cd poky
+git clone https://git.yoctoproject.org/meta-intel --branch langdale
+git clone https://git.openembedded.org/meta-openembedded --branch langdale
+git clone https://github.com/kraj/meta-clang.git
```

Comment:
missing the branch for the last line?

Reply:
Yes and no. This is the way it should be, meaning the master from meta-clang should be cloned.